src/mpcCoreKit.ts

```diff
@@ -175,7 +175,7 @@ export class Web3AuthMPCCoreKit implements ICoreKit {
     return this.options.uxMode === UX_MODE.REDIRECT;
   }
 
-  public async init(): Promise<void> {
+  public async init(params: { handleRedirectResult: boolean } = { handleRedirectResult: true }): Promise<void> {
```

Comment:
should define type for `{ handleRedirectResult: boolean }` as this is part of the external interface. Or just use `handleRedirectResult` as single parameter...